features/step_definitions/features_steps.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+Given('a feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+end
+
+Given('an enabled feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+  @service_plan.features << feature
+  @service_plan.save!
+end
+
+Then('there {is} no feature named {string}') do |is, string|  
+  feature_name = @service_plan.service.features.last.name
```

Comment:
Why "last" here? What happens if there are more than 1 features on the service?

Also, "is" argument is not used.

And if you wanted to use `{is}` parameter type, you could actually merge the two steps, no need to separate them, because it would match both "there is no feature" and "there is feature", and the variable `is` would be a boolean with `true` or `false`